### DISCUSSION TITLE
Journal Aria

### DISCUSSION THREAD

**Discussion by Ripus28** [Dilomos/aria_foundryvtt#56] [General]
Bonjour,
j'ai l'impression que depuis la dernière release, le CSS a été modifié au niveau des journaux. On ne vois plus apparaître les écrous quand on utilise "code" Les Citations non plus ne fonctionne plus avec le fond bleu et la bulle etc... Est-ce normal ?

**Comment by Dilomos** [comment]
Bonjour,

Alors oui c'est normal, mais ça n'a pas complétement disparu.

La personnalisation du CSS pour le journal a été ajouté dans la V2.2.0 par PixelReveur, mais elle modifiait le style des classes de l'éditeur de Foundry VTT qui peuvent changer selon les mise a jour du logiciel.
Pour être plus indépendant j'ai préféré recréer des classes CSS dédié pour Aria, le seul petit problème étant qu'actuellement avec la V10, on ne peut pas modifier facilement l’éditeur 'prosemirror' utilisé dans le journal et donc lui ajouté nos propre classes.

La solution, en attendant de pouvoir modifier prosemirror (des discussions sont en cours sur le discord de Foundry), c'est d'utiliser l'ancien editeur TinyMCE qui est toujours présent :
<img width="1071" alt="StyleTinyMCE" src="https://user-images.githubusercontent.com/49794325/193992482-80622f6b-37a8-4563-b9cc-2d8eeb52b87a.png">

En changeant la "Sheet" de la page qu'on édite on peut choisir TinyMCE, et la toute les classes personnalisée d'Aria apparaissent dans le menu déroulant.
Je conseillerai également de bien mettre le système à jour en v2.6.2 pour avoir le CSS le plus à jour (j'ai fait pas mal de modification hier)

**Reply by Dilomos** [reply]
Également, une fois les classes ajoutée il est tout a fait possible de rebasculer sur l'editeur par defaut (prosemirror) car le code HTML créé par TiniMCE contient les classes et du coups le CSS s'applique

**Comment by Ripus28** [comment]
Super,
merci beaucoup, vraiment TOP :)
bravo pour tout ce travail !
